### DISCUSSION TITLE
Single Page Application for libraries

### DISCUSSION THREAD

**Discussion by trois-six** [gotson/komga#1682] [Ideas]
**Describe your suggested feature**

When I browse a library, for the moment I do have to click on "ALL", #, A, B... or 1, 2, 3... to go the next page.

Could we go automatically to the next page when we scroll down? With a vertical selector to go faster directly to a letter? The current user experience is not very practical when you want to look at all the contents of your library.

**Comment by gotson** [comment]
There used to be load on scroll before, it was removed because it makes the navigation extremely difficult. Various filters as well as search is the way to go.
